examples/angular/src/pages/ui/components/authenticator/custom-sign-up-fields/aws-exports.js

```diff
@@ -0,0 +1,2 @@
+import awsExports from '@environments/auth-with-email-and-custom-attributes/src/aws-exports';
+export default awsExports;
```

Comment:
I assume symlinking didn't work and instead these had to be explicit exports?

Also, I think this works too:

```js
export * from '@environments/auth-with-email-and-custom-attributes/src/aws-exports';
```

Or
```
export default from '@environments/auth-with-email-and-custom-attributes/src/aws-exports';
```